index.js

```diff
@@ -850,8 +857,14 @@ Command.prototype.description = function(str) {
  */
 
 Command.prototype.alias = function(alias) {
-  if (0 == arguments.length) return this._alias;
-  this._alias = alias;
+  var command = undefined;
+  if(this.commands.length == 0) {
+    command = this;
+  } else {
+    command = this.commands[this.commands.length - 1]
```

Comment:
@zhiyelee Thanks for the review, You should fix test in `test.command.executableSubcommandAlias.js`, I changed test as below and passed the test,
![image](https://cloud.githubusercontent.com/assets/3284018/16551275/4740df0a-41f1-11e6-875c-d4616af48604.png)